test/synapse/2019-06-01/azartifacts/pollers_helper.go

```diff
@@ -0,0 +1,768 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is regenerated.
+
+package azartifacts
+
+// TODO replace this code with data-plane specific definitions
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+)
+
+const (
+	headerAsyncOperation = "Azure-AsyncOperation"
+	headerLocation       = "Location"
+)
+
+const (
+	operationInProgress string = "InProgress"
+	operationCanceled   string = "Canceled"
+	operationFailed     string = "Failed"
+	operationSucceeded  string = "Succeeded"
+)
+
+var pollingCodes = [...]int{http.StatusNoContent, http.StatusAccepted, http.StatusCreated, http.StatusOK}
+
+type pollingTracker interface {
```

Comment:
This is all the ARM stuff.  Can we have this file generate a little as possible?  Once we start generating real content the diff will be extremely noisy.

Reply:
done